test/System/Workflow/PerforatedScreenByImage/PanelWall.dyn

```diff
@@ -1,4 +1,4 @@
-<Workspace Version="0.8.1.1310" X="-329.867792183962" Y="-810.6891383816" zoom="0.653999019844256" Name="Home" RunType="Automatic" RunPeriod="100" HasRunWithoutCrash="True">
+<Workspace Version="0.8.1.1310" X="-329.867792183962" Y="-810.6891383816" zoom="0.653999019844256" Name="Home" RunType="Manual" RunPeriod="100" HasRunWithoutCrash="True">
```

Comment:
Any reason for the change in RunType? we want to ensure that our tests run in automatic execution mode.